src/OSPSuite.UI/Binders/ArithmeticMeanAreaCurveBinder.cs

```diff
@@ -14,9 +14,9 @@ public ArithmeticMeanAreaCurveBinder(Curve curve, ChartControl chartControl, Cur
       {
       }
 
-      protected override bool AddRelatedValuesToRow(DataRow row, DataColumn yData, IDimension yDimension, Unit yUnit, double y, float baseValue)
+      protected override bool AddRelatedValuesToRow(DataRow row, DataColumn yData, IDimension yDimension, Unit yUnit, double y, int baseIndex)
       {
-         var stdDev = yDimension.BaseUnitValueToUnitValue(yUnit, Curve.yData.GetValue(baseValue));
+         var stdDev = yDimension.BaseUnitValueToUnitValue(yUnit, Curve.yData.GetValueAt(baseIndex));
```

Comment:
GetValue calls baseGrid.IndexOf underneath.
We can bypass that if we have the index

Reply:
The code is absolutely not equivalent. GetValue interpolates while GetValueAt just gets. 
I am not sure if, in this specific instance, this makes a difference or not. Hard to say. But this is not equivalent code